dashboards/sentry-software/sentry-software.json

```diff
@@ -0,0 +1,3021 @@
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountIds": [],
+                  "query": "SELECT min(`hw.host.heating_margin`) as '°C' FROM Metric WHERE site IN ({{sentry_site}}) FACET site SINCE 1 week AGO UNTIL NOW TIMESERIES 1 hour"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "units": {
+                "unit": "CELSIUS"
+              },
+              "yAxisLeft": {
+                "max": 40,
+                "min": 0,
+                "zero": false
+              },
+              "yAxisRight": {
+                "zero": true
+              }
+            }
+          }
+        ]
+      },
+      {
+        "name": "Site",
+        "description": null,
+        "widgets": [
+          {
+            "title": "",
+            "layout": {
+              "column": 1,
+              "row": 1,
+              "width": 2,
+              "height": 2
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![Sentry Software Logo](https://dl.sentrysoftware.com/public/otel-dl-link.png)\n\n#### IMPORTANT: Select a site from the variable dropdown to facet the widgets by site."
+            }
+          },
+          {
+            "title": "Monitoring Coverage",
+            "layout": {
+              "column": 3,
+              "row": 1,
+              "width": 2,
+              "height": 2
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountIds": [],
+                  "query": "FROM(FROM Metric \n  SELECT filter( latest(`hardware_sentry.connector.status`), WHERE state = 'ok' AND host.name IN ({{sentry_host}}) AND site IN ({{sentry_site}}) ) as 'status'\n  FACET site,host.id LIMIT 200) SELECT sum (`status`)*100/(FROM Metric SELECT cardinality(hardware_sentry.host.configured) SINCE 24 hours ago WHERE site IN ({{sentry_site}})) as '% coverage' SINCE 24 hours ago"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "thresholds": [
+                {
+                  "alertSeverity": "WARNING",
+                  "value": 90
+                },
+                {
+                  "alertSeverity": "CRITICAL",
+                  "value": 50
```

Comment:
Can we configure the alert to be triggered if it exceeds 90% (critical) and 85% (warning) threshold?

Reply:
Same thing as above actually, this one is not really an alert even though it says alert severity. It's used for dashboard UI, and I believe the thresholds are correct here.